autopilot/hostfilter.go

```diff
@@ -90,7 +90,7 @@ func isOutOfFunds(cfg Config, h Host, c worker.Contract) bool {
 }
 
 func isUpForRenewal(cfg Config, c rhpv2.ContractRevision, blockHeight uint64) bool {
-	return blockHeight+cfg.Contracts.RenewWindow/2 >= c.EndHeight()
+	return blockHeight+cfg.Contracts.RenewWindow >= c.EndHeight()
```

Comment:
hm this does make sense but we probably have to rethink the current contract set because contracts up for renewal are excluded from the contract set currently, we might have to reconsider excluding them from the contract set. We had a `TODO` there anyway at one point indicating it was quite harsh.